Components/Typography/typography.css

```diff
@@ -34,24 +35,67 @@
   
 }
 
-.font-extra-large{
-    font-size: 5rem;
+
+
+.h1{
+    color: var(--floating-btn-background);
+    font-size: 4rem;
+    font-weight: 700;
+}
+.h2{
+    color: var(--floating-btn-background);
+    font-size: 3.5rem;
+    font-weight: 700;
+}
+.h3{
+    color: var(--floating-btn-background);
+    font-size: 3rem;
+    font-weight: 700;
+}
+.h4{
+    color: var(--floating-btn-background);
+    font-size: 2.5rem;
+    font-weight: 700;
+}
+.h5{
+    color: var(--floating-btn-background);
+    font-size: 2rem;
+    font-weight: 700;
+}
+.h6{
+    color: var(--floating-btn-background);
+    font-size: 1.5rem;
+    font-weight: 700;
 }
 
+.line-through{
+    text-decoration:line-through;
+}
 
-.font-large{
-    font-size:4rem;
+.text-underline{
+    text-decoration: underline;
 }
 
 
-.font-mid{
-    font-size: 2rem;
+.bold-text{
+    font-weight: 900;
 }
 
-.font-small{
-    font-size: 1.5rem;
+.normal-text{
+    font-weight: 400;
 }
 
-.font-extra-small{
-    font-size: 1rem;
+.light-text{
+    font-weight: 100;
+}
+
```

Comment:
Your normal and light text looks same, I believe font-weight is not working right for some reason.

Reply:
I played with font-weight specifically using normal, lighter and 100 using playit feature in w3schools, and in all these 3 cases the text looks the same to me too. I don't know if it's just me who's not able to notice the 
difference. @Naman-Saxena1